dealership/contracts.py

```diff
@@ -1,12 +1,48 @@
 class Contract(object):
-    pass
-
+    def __init__(self, customer, vehicle):
+        self.customer = customer
+        self.vehicle = vehicle
+        self.interest = {
+            'Car': 1.07,
+            'Motorcycle': 1.03,
+            'Truck': 1.11
+        }
```

Comment:
I would maybe consider making the interest rates part of the respective vehicle classes.